mlflow/server/js/src/components/MetricsPlotControls.js

```diff
@@ -23,6 +23,9 @@ export class MetricsPlotControls extends React.Component {
     handleYAxisLogScaleChange: PropTypes.func.isRequired,
```

Comment:
The main updates in this component (which renders the plot controls e.g. the line smoothness slider) are to pass plot configs as props, so that we can initialize e.g. the line smoothness slider with the smoothness specified in the plot URL, toggle the y-axis-log-scale switch if a log-scale y axis is enabled, etc.